mmv1/products/sql/Database.yaml

```diff
@@ -48,6 +48,7 @@ async:
 collection_url_key: 'items'
 custom_code:
   pre_delete: 'templates/terraform/pre_delete/sql_database_deletion_policy.tmpl'
+  pre_read: 'templates/terraform/pre_read/sql_database_activation_policy.tmpl'
```

Comment:
With the other test you've made as an example, could you please make a similar acceptance test that ensures that the Read action on the database resource is disabled as expected when the instance is availability_type = "NEVER"? It would essentially be a copy/paste of the test you've already added but with the database resource present in the config instead.